app/scripts/bullets/bullet.directive.js

```diff
@@ -10,6 +10,7 @@ bulletApp.directive('bullet', function () {
             header: '@'
         },
         link: function (scope, element) {
```

Comment:
I'd say there's a good bit of logic going on in this directive that isn't directly related to managing the scope.